packages/lib/modules/tokens/TokenIcon.tsx

```diff
@@ -76,7 +76,7 @@ export function TokenIcon({
           borderRadius="100%"
           height={`${size}px`}
           onError={() => !hasError && setHasError(true)}
-          src={hasError || !iconSrc ? fallbackSVG.toDataUriSync() : iconSrc}
+          src={hasError || !iconSrc ? fallbackSVG.toDataUri() : iconSrc}
```

Comment:
This doesn't work. `toDataUri` returns a promise, we need to use `toDataUriSync` or find another way to do this.

Reply:
Did you run pnpm install? Did you see this comment? 
https://github.com/balancer/frontend-monorepo/pull/335#issuecomment-2556846355